paper-badge.html

```diff
@@ -79,7 +98,10 @@
       }
     </style>
 
-    <div id="badge">{{label}}</div>
+    <div class="badge">
+      <iron-icon hidden$="{{!isIconBadge}}" icon="{{icon}}"></iron-icon>
+      <span id="badge-text" hidden$="{{isIconBadge}}">{{label}}</span>
```

Comment:
Given your comment below, then you can still use a computed function for `hidden` -- it just really doesn't need to be a property. Like this: https://github.com/PolymerElements/paper-checkbox/blob/master/paper-checkbox.html#L199